test/parallel/test-os.js

```diff
@@ -93,8 +94,9 @@ const release = os.release();
 is.string(release);
 assert.ok(release.length > 0);
 //TODO: Check format on more than just AIX
-if (common.isAix)
+if (common.isAix) {
```

Comment:
The only rule we currently have is to use braces when the contained statement is more than one line. I don't think it is necessary to enforce either behavior when braces are optional, but if it does not churn too much, we might as well lint it, even though I don't think we will get consensus on that.